### ISSUE TITLE
In-Game: flexpanel_node_get_parent() causes silent crash if called on the root node in 2024.800 Betas

### ISSUE THREAD

**Issue by yunzlp** [YoYoGames/GameMaker-Bugs#6996]
### Description

Calling flexpanel_node_get_parent() on the root node will cause a silent crash.

E.g.
```
resultString = flexpanel_node_get_parent(flexpanel_create_node());
```

### Expected Change

Return "undefined", so the game does not crash

### Steps To Reproduce

1. Start GameMaker and import [flexpanelnodeparentcrash.zip](https://github.com/user-attachments/files/16425126/flexpanelnodeparentcrash.zip)
2. Run it (e.g., Windows VM)
3. Observe you don't see the game window, as the crash happens on startup in this project
4. See the Output Window

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.800 (Betas)

### Which platform(s) are you seeing the problem on?

Windows

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [x] I have included a small sample project

**Comment by YYDan** [comment]
Made a proper project for you

**Comment by rwkay** [comment]
Fixed in 2024.8

**Comment by Emc1923** [comment]
Verified in  IDE v2024.800.0.605  Runtime v2024.800.0.628.
